src/types/chartdata.raw.ts

```diff
@@ -20,7 +22,9 @@ export const Chartdata = z.object({
   voteEnd: z.number(),
   totalVotes: z.number(),
   bribedVotes: z.number(),
+  externallyBribedVotes: z.number(),
```

Comment:
please make it '.optional()' to avoid conflicts with old data, that doesn't have this data